appcues/src/main/java/com/appcues/debugger/screencapture/ViewExt.kt

```diff
@@ -75,3 +77,11 @@ internal fun View.asCaptureView(): Capture.View? {
         children = children,
     )
 }
+
+internal fun View.screenshot() =
```

Comment:
we could move these specific methods inside that new class that would only work on capturing the screen.